src/sfizz/Defaults.h

```diff
@@ -124,6 +124,41 @@ namespace Default
 	constexpr float rtDecay { 0.0f };
 	constexpr Range<float> rtDecayRange { 0.0f, 200.0f };
 
+    // Performance parameters: Filters
+    constexpr float filterCutoff { 0 };
+    constexpr float filterResonance { 0 };
+    constexpr int filterKeytrack { 0 };
+    constexpr uint8_t filterKeycenter { 60 };
+    constexpr int filterRandom { 60 };
+    constexpr int filterVeltrack { 0 };
+    constexpr int filterCutoffCC { 0 };
+    constexpr float filterResonanceCC { 0 };
+    constexpr Range<float> filterCutoffRange { 0.0f, 96000.0f };
+    constexpr Range<int> filterCutoffModRange { -9600, 9600 };
+    constexpr Range<int> filterKeytrackRange { 0, 1200 };
+    constexpr Range<int> filterRandomRange { 0, 9600 };
+    constexpr Range<int> filterVeltrackRange { -9600, 9600 };
+    constexpr Range<float> filterResonanceRange { 0, 40 };
```

Comment:
Not that it matters a lot, but here literals want to have ".0f" to be consistent with the rest of style.